tensorboard/webapp/BUILD

```diff
@@ -265,6 +265,7 @@ tf_ng_web_test_suite(
         "//tensorboard/webapp/core/views:test_lib",
         "//tensorboard/webapp/customization:customization_test_lib",
         "//tensorboard/webapp/deeplink:deeplink_test_lib",
+        "//tensorboard/webapp/feature_flag/effects:effects_test_lib",
```

Comment:
feature_flag already has its own test suite so no need to add them here:
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/feature_flag/BUILD;l=61;drc=28ff87e12c60fe1daafb9f17c0cc86d23a04e79d

At some point in 2011 Stephan started asking us to put new tests in their own karma test suites. I believe he realized that there was still some benefit to parallelizing the jobs depiste the "overhead of bootstrap/tearDown" that he mentions in the comment of L248/249.